doc/man5/fips_config.pod

```diff
@@ -0,0 +1,65 @@
+=pod
+
+=head1 NAME
+
+OPENSSL FIPS CONFIGURATION
+
+=head1 DESCRIPTION
+
+A separate configuration file containing data related to FIPS 'self tests' is
+written to during installation time.
+This data is used for 2 purposes when the fips module is loaded.
+
+ - Verify the module's checksum each time the fips module loads.
+ - Run the startup FIPS self test KATS (known answer tests). This only needs
```

Comment:
Line break after period, please.

Also, if you want to make a list POD-style, use POD syntax:

```
This data is used for 2 purposes when the fips module is loaded:

=over 4

=item - Verify the module's checksum each time the fips module loads.

=item - Run the startup FIPS self test KATS (known answer tests).
This only needs to be run once during installation.

=over
```